settings.gradle

```diff
@@ -74,6 +74,7 @@ include ':test:smoke:testApps:HttpClients'
 include ':test:smoke:testApps:WebAuto'
 include ':test:smoke:testApps:SpringBoot1_3Auto'
 include ':test:smoke:testApps:SpringBootAuto'
+include ':test:smoke:testApps:SpringScheduler'
```

Comment:
we need to remember to update CI's smoke test job list after merging this

Reply:
yes, i remember :)